prow/manifests/overlays/metal3/config.yaml

```diff
@@ -733,7 +733,7 @@ presubmits:
         - main
         - release-1.3
         - release-1.4
-      run_if_changed: '^api|^test|^Makefile$'
+      run_if_changed: '^api|^test|^Makefile$|^go.mod$|^main\.go$'
```

Comment:
Maybe better to change it to `skip_if_only_changed`?
```suggestion
      skip_if_only_changed: '(((^|/)OWNERS)|(\.md))$'
```
Otherwise we won't build on changes to `controllers/` or `baremetal/`

Reply:
Sure, fixed.